content/en/continuous_integration/guides/rum_integration.md

```diff
@@ -19,13 +24,21 @@ The CI Visibility - RUM integration is only available for these versions of `cyp
 
 ### Browser tests and RUM
 
-If you use Cypress for your browser tests and the application being tested is instrumented using [RUM][1], your test results and their generated RUM browser sessions and session replays are automatically linked. A new **Browser Sessions** tab appears in the test detail side panel:
+**Important**: Before you can link your Cypress tests and RUM, make sure that [Test Visibility][1] is already setup for Cypress.
```

Comment:
Non-blocking, if this is something you want to highlight to users, consider an alert div, similar to https://a.cl.ly/7KuWqqew:
```suggestion
<div class="alert alert-warning">
<strong>Important</strong>: Before you can link your Cypress tests and RUM, make sure that <a href="/continuous_integration/tests/javascript/?tab=cypress#instrument-your-tests">Test Visibility</a> is already setup for Cypress.</div>
```